src/applications/travel-pay/components/submit-flow/pages/ConfirmationPage.jsx

```diff
@@ -36,10 +36,10 @@ const ConfirmationPage = () => {
           <h2 slot="headline">Claim submitted</h2>
           <p className="vads-u-margin-y--0">
             This claim is for your appointment at{' '}
-            {data.location.attributes.name}{' '}
-            {data.practitioners.length > 0
-              ? `with ${data.practitioners[0].name.given.join(' ')} ${
-                  data.practitioners[0].name.family
+            {data.location?.attributes?.name}{' '}
+            {data.practitioners?.length > 0
+              ? `with ${data.practitioners[0]?.name?.given?.join(' ')} ${
```

Comment:
Are we thinking that `data.practitioners[0]?` protects against cases like `practitioners: [null, 0, false, !!!'', etc]`?

Maybe worth filtering out falsey values from the practitioners list beforehand?

Reply:
Is there ever a case where the `given` name is not present/falsey?

Reply:
or actually, is there ever a case where the `name` is not present/falsey?